src/legacy/core_plugins/vis_type_markdown/public/_markdown_vis.scss

```diff
@@ -2,3 +2,13 @@
   padding: $euiSizeS;
   width: 100%;
 }
+
+.visEditor--markdown {
+  vis-editor-vis-options, vis-options-react-wrapper, textarea {
+    @include flex-parent(1, 1, auto);
```

Comment:
This mixin is old legacy code, we'd like to move away from using it. Can you change this block to:

```scss
.visEditor--markdown {
  vis-editor-vis-options, vis-options-react-wrapper {
    flex-grow: 1;
    display: flex;
    flex-direction: column;
  }

  .visEditor--markdown__textarea { // Add this class to the EuiTextArea component
    flex-grow: 1;
  }

  .mkdEditor {
    height: 100%;
  }
}
```

Reply:
Updated. Thanks!